deepmd/dpmodel/descriptor/se_t.py

```diff
@@ -0,0 +1,362 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+import itertools
+
+import numpy as np
+
+from deepmd.dpmodel.utils.update_sel import (
+    UpdateSel,
+)
+from deepmd.env import (
+    GLOBAL_NP_FLOAT_PRECISION,
+)
+from deepmd.utils.path import (
+    DPPath,
+)
+from deepmd.utils.version import (
+    check_version_compatibility,
+)
+
+try:
+    from deepmd._version import version as __version__
+except ImportError:
+    __version__ = "unknown"
```

Comment:
Unused import.

Reply:
I noticed that unused import of `__version__` exists **several files** in deepmd/dpmodel. Maybe I use another PR to fix them after this PR merged.